packages/core/src/routes/sign-in-experience.test.ts

```diff
@@ -25,38 +25,37 @@ describe('signInExperiences routes', () => {
   });
 
   it('PATCH /sign-in-ex/:id', async () => {
-    const companyInfo = {
-      name: 'silverhand',
-      logo: 'http://silverhand.png',
-    };
-    const signInMethods = {
-      primary: ['phone'],
-      secondary: ['email'],
-      disabled: [],
+    const branding: Branding = {
+      primaryColor: '#000',
+      backgroundColor: '#fff',
+      darkMode: true,
+      darkBackgroundColor: '#000',
+      darkPrimaryColor: '#fff',
+      style: BrandingStyle.Logo,
+      logoUrl: 'http://silverhand.png',
+      slogan: 'silverhand',
     };
 
+    const socialSignInConnectorIds = ['abc', 'def'];
+
     const response = await signInExperienceRequester.patch('/sign-in-ex/default').send({
-      companyInfo,
-      signInMethods,
+      branding,
+      socialSignInConnectorIds,
     });
 
     expect(response.status).toEqual(200);
     expect(response.body).toEqual({
       ...mockSignInExperience,
-      companyInfo,
-      signInMethods,
+      branding,
+      socialSignInConnectorIds,
```

Comment:
I checked koaGuard for this PATCH /sign-in-ex/:id route, the body can be partially updated, so maybe add a test case for partially updating sign-in-exp in DB or simply remove `...mockSignInExperience` is good enough to cover all cases.

Reply:
It's a partial update.

`...mockSignInExperience` is just used to match the returning complete data after updating, so it should not be removed.